db/violet_seeds/violet.rb

```diff
@@ -0,0 +1,1201 @@
+}
+.navbar-toggler span:nth-child(1) {
+  transform: translate(0%, 0%) rotate(0deg);
+}
+.navbar-toggler span:nth-child(2) {
+  opacity: 1;
+}
+.navbar-toggler span:nth-child(3) {
+  transform: translate(0%, 0%) rotate(0deg);
+}
+.btn-demo {
+  padding: 9px 15px;
+}
+.navbar-collapse > hr {
+  margin: 1rem 0 0.5rem;
+  border-top: 2px solid #6f5afe;
+  opacity: 0.2;
+}
+.jumbo-top {
+  padding-top: 35px;
+}
+.bg-grey {
+  background-color: #edebff;
+}
+.embed-responsive {
+  border-radius: 10px;
+}
+.product-img {
+  display: block;
+  transition: all 0.6s ease-out;
+}
+.product-img:hover {
+  transform: scale(1.05);
+}
+.products .product-img img {
+  max-width: 100%;
+  height: auto;
+  object-fit: contain;
+}
+.img-div {
+  padding-left: 0;
+}
+@media (max-width: 992px) {
+  header.container {
+    padding: 0 6px;
+  }
+}
+@media (min-width: 992px) {
+  .navbar-expand-lg .navbar-nav .nav-link {
+    padding-right: 20px;
+    padding-left: 20px;
+  }
+}
+@media (max-width: 767.98px) {
+  .badge {
+    font: normal normal 800 14px/17px Karla;
+    padding: 5px 10px;
+  }
+  h1 {
+    font: normal normal 800 46px/54px Karla;
+  }
+  h2 {
+    font: normal normal 800 28px/33px Karla;
+  }
+  h3 {
+    font: normal normal 800 26px/31px Karla;
+  }
+  h5 {
+    font: normal normal 800 22px/26px Karla;
+  }
+  .btn-lg {
+    font: normal normal bold 14px/17px Karla;
+    padding: 15px 25px;
+  }
+  .jumbo-top {
+    padding-top: 0;
+  }
+  .benefits .card {
+    margin-left: 0 !important;
+    margin-right: 0 !important;
+  }
+  .card-header > .img-fluid {
+    max-width: 30px;
+  }
+  .get-started {
+    padding: 30px;
+  }
+  .get-started > p {
+    max-width: none;
+  }
+  .get-started a {
+    white-space: nowrap;
+    padding: 15px 20px;
+  }
+  .img-div {
+    padding: 0;
+  }
+}
+.footer-bottom {
+  background: #e0ddfb;
+}
+.footer-logo {
+  max-width: 121px;
+}
+.footer-bottom .logo {
+  max-width: 21px;
+}
+.nav-link.current {
+  color: #6f5afe;
+}
+.cursor-pointer {
+  cursor: pointer;
+}
+.navbar-brand {
+  margin-right: 10px;
+}
+</style><main class="main">
+<div class="container">
+	<div class="jumbotron jumbo-top bg-transparent px-0 mb-0">
+		<div class="row d-flex justify-content-center align-items-center">
+			<div class="col-xl-6 flex-column d-flex justify-content-center align-items-start">
+				<div>
+					<div class="mb-2">
+						<span class="badge badge-primary bg-transparent text-danger">VIOLET RAILS</span>
+					</div>
+					<h1 class="text-capitalize">A Feature Rich <br><span class="text-accent">Web Platform</span><br>
+					</h1>
+					<p class="lead mt-3"> For hosting your next big idea
+					</p>
+				</div>
+				<div class="d-flex align-items-center my-5">
+					<a href="/forum" class="btn btn-lg btn-primary btn-shadow">Forum</a>
+					<a href="https://github.com/restarone/violet_rails" class="d-flex align-items-center btn btn-lg font-weight-bold bg-transparent">
+					<img src="{{ cms:file_link 31 }}" class="img mr-2">
```

Comment:
@yashkir where we use tokens like `{{ cms:file_link 31 }}` how are we ensuring these hard coded ID's (eg: `31`) match what is in the file system?